### ISSUE TITLE
[Request] Autocomplete return a data table

### ISSUE THREAD

**Issue by PauloFerreira25** [quasarframework/quasar#2119]
Hello, gentlemen.
Please, could you implement an autocomplete template that returns a data table in place of a list?
I say this, because the possible customizations to be made in the data table are much larger than in the list.
I tried doing it myself in the source code of the quasar, but my ability is very low.

I gave an example of what the visual would look like:
![captura de tela de 2018-06-02 17-42-14](https://user-images.githubusercontent.com/1591772/40880721-62b19114-668c-11e8-9828-9c12c47d706e.png)
PS: I could not adjust the popover size

Below I am putting the code I made to set up the example. But I do not think it's very useful.
[https://gist.github.com/PauloFerreira25/0c6bfceddb3b8f1b571d01c2edd6ff20](url)

In the system I'm doing, I already have 2 practical examples of how this structure would be better:
Search Users
Search Products

Today, the only way I can do this is by using a button in the form that opens a modal containing the table, so that the selection of the table returns to the form.

Sorry for the google translator english

I hope you meet my request. For me, quasar is the best framework of today. You are the best.

**Comment by rstoenescu** [comment]
Hi,

Quasar offers the foundation (the building blocks) on which you can build this yourself. It makes no sense to clobber this custom implementation in the framework. If you need help with it then require it on Discord or forum. Thanks.

**Comment by PauloFerreira25** [comment]
Hi, thank you so much for replying.
I think I expressed myself poorly.
Quasar includes a screen like mine, I am asking you to include the return of a generic table that can be modified.

I understand that it is possible to create using the "blocks" of the quasar. was as I set the example.

However, some implementations that you have already done, I would have to redo.
Example:
You have already done the autocomplete control of lazy, with the loading
You have already made the box size contole, to come out equal to the size of the input.
You have already clicked, be even in v-model.
You already control the creation and destruction of elements in html.

Among other functions.
I'm not saying that doing it on your own is impossible. Just believe, that having something like this ready, is an item that would facilitate the development of complex systems.

Thank you very much for your attention.
